src/rules/completedDocsRule.ts

```diff
@@ -307,7 +307,22 @@ const modifierAliases: { [i: string]: string } = {
     export: "exported",
 };
 
-function walk(context: Lint.WalkContext<ExclusionsMap>, typeChecker: ts.TypeChecker) {
+/**
+ * @remarks See https://github.com/ajafff/tsutils/issues/16
+ */
+const getApparentJsDoc = (node: ts.Node): ts.JSDoc[] => {
+    if (node.kind === ts.SyntaxKind.VariableDeclaration) {
+        const parent = node.parent as ts.VariableDeclarationList;
+
+        return node === parent.declarations[0]
+            ? tsutils.parseJsDocOfNode(parent)
+            : tsutils.parseJsDocOfNode(node);
```

Comment:
this is not what you want to do.

there are a few things to consider here:
* if the `VariableDeclaration` is not the first of the declaration list, TypeScript won't attach any JSDoc to that variable. Neither should this rule.
* If you have a `VariableDeclarationList`, you need to get the JSDoc from the containing `VariableStatement`. `parseJsDocOfNode` will fail if you have `/**foo*/ export let foo;` for example.

You may want to look at `deprecationRule.ts:200`. We already do something similar there.